tests/lib/prepare-uc20.sh

```diff
@@ -0,0 +1,225 @@
+#!/bin/bash
+
+set -e
+set -x 
+
+apt update -yqq
```

Comment:
perhaps it is not needed, I don't see any apt use after

Reply:
Good point